src/module/Ui/templates/authentication/activate.twig

```diff
@@ -0,0 +1,9 @@
+{% do placeholder('hideSubjectNav').set('True') %}
```

Comment:
Yeah, you're right. I feel ashamed that I thought PHP would care what type I pass it ;)